README.rst

```diff
@@ -30,24 +30,24 @@ Installation & basic usage
 
     .. code:: python
 
-        from opencensus.trace import request_tracer
+        from opencensus.trace import tracer as tracer_module
```

Comment:
Yes, this looks good.  I always struggle with the "package/module/class/variable" naming, since I have up to four things but only two ways (Uppercase/lowercase) to name them.

Reply:
:+1: